psfr/psfr.py

```diff
@@ -403,6 +413,48 @@ def centroid_fit(data, model, mask=None, variance=None, oversampling=1):
         required shift in units of pixels in the data such that the model matches best the data
     """
 
+    def getProbability(pars, variance, mask, oversampling, model, data):
```

Comment:
this definition is basically the same (modulo a minus sign) fo the _minimise() function that is already implemented, so no need to duplicate the code here